lib/mayaUsd/render/vp2ShaderFragments/usdPreviewSurfaceCombiner.xml

```diff
@@ -87,12 +101,31 @@ usdPreviewSurfaceCombiner(
     result.outColor += diffuseColor * irradianceEnv;
```

Comment:
Can you try:
```
if (useSpecularWorkflow) {
    result.outColor += diffuseIrradianceIn;
    result.outColor += diffuseColor * irradianceEnv;
} else {
    // For metallic workflows, pure metals have no diffuse
    result.outColor += diffuseIrradianceIn * (1.0 - metallic);;
    result.outColor += diffuseColor * irradianceEnv * (1.0 - metallic);
}
```
To see if it helps? Might remove the need to modulate by `length(irradianceEnv)` later.
Ref: https://github.com/PixarAnimationStudios/USD/blob/dev/pxr/usdImaging/plugin/usdShaders/shaders/previewSurface.glslfx#L316

Reply:
I think this definitely helps. For comparison, the before and after. The metallic portions look a lot better on the assets I've tested with.

![image](https://user-images.githubusercontent.com/4443090/115909268-2913bc00-a420-11eb-9e40-131068d10d43.png)
![image](https://user-images.githubusercontent.com/4443090/115909306-33ce5100-a420-11eb-8996-97328dc82e8d.png)